starboard/BUILD.gn

```diff
@@ -311,3 +312,15 @@ if (current_toolchain == starboard_toolchain) {
     }
   }
 }
+
+test("starboard_unittests") {
+  testonly = true
+
+  sources = [ "//starboard/common/log_unittest.cc" ]
+
+  public_deps = [
```

Comment:
shouldn't starboard_unittest depend on the target //starboard:starboard_group($starboard_toolchain") (the starboard target) or are you just testing some parts of ?

Another thing : If im reading the thread correctly these tests are for "below" starboard targets so should be built with starboard toolchain https://chat.google.com/room/AAAAmC3z4d8/WeXyq7X45yE/WeXyq7X45yE?cls=10

This test needs to be built from an existing target we build in CI: add it to gn_all target in cobalt/BUILD.gn 
Also when you build the target make sure it's being built with starboard toolchain(assuming my above comment is correct) . An example is here: https://source.corp.google.com/h/github/youtube/cobalt/+/main:cobalt/BUILD.gn;l=22?q=cobalt%2FBUILD.gn&ss=h%2Fgithub%2Fyoutube%2Fcobalt%2F%2B%2Frefs%2Fheads%2Fmain

Reply:
also from the comments, it's seems like the test is intended more for 3P platforms , if that's the case you'll need to add an if(is_starboard) : so that the test doesn;t run unintentionally on paltforms which don't support starboard as a platform(android)

Reply:
I'm a bit confused in this new chrobalt world.

Some starboard tests will be applicable to android, some won't- things which involve SbEventHandle/ some other not used parts of starboard - tests will probably need to be filtered by use_starboard_media / is_android/ is_starboard at some point .  

A proposed default(which im making up right now) is to add tests without any flag and then if they fail for android - consider those case by case and filter those test cases out using one of the above flags(whatever is appropriate)